community.html

```diff
@@ -280,6 +289,19 @@ <h2> Dynamic Tools </h2>
         <li> Source: <a href="https://github.com/ucd-plse/HiFPTuner">github.com/ucd-plse/HiFPTuner</a> </li>
     </ul> </dd>
 
+    <dt>pLiner</dt>
+
+    <dd> pLiner is a tool for automatically identifying code that will trigger
+    compiler-induced numerical variability. It works by combining selective
+    precision enhancements and a guided dynamic search. The authors used the
+    tool to identify a particular inconsistency within minutes that took weeks
+    to diagnose without the tool. </dd>
```

Comment:
```suggestion
    precision enhancements and a guided dynamic search. This can make it a lost faster to identify numerical issues. </dd>
```

I think the existing text was too vague for most readers.